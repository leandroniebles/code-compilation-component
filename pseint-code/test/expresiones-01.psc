Proceso expresiones
	Escribir 1+20+300+4000;
	Escribir 1-20+300+4000;
	Escribir 1-20-300+4000;
	Escribir 1-20-300-4000;
	Escribir 1+20-300-4000;
	Escribir 1+20+300-4000;
	Escribir 1-20+300-4000;
	Escribir 1+20-300+4000;
	Escribir 1+20+300+4000;
	Escribir -1-20+300+4000;
	Escribir -1-20-300+4000;
	Escribir -1-20-300-4000;
	Escribir -1+20-300-4000;
	Escribir -1+20+300-4000;
	Escribir -1-20+300-4000;
	Escribir -1+20-300+4000;
FinProceso
