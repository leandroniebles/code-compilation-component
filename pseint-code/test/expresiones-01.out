4321
4281
3681
-4319
-4279
-3679
-3719
3721
4321
4279
3679
-4321
-4281
-3681
-3721
3719
